CONTRIBUTING.md

```diff
@@ -169,34 +159,50 @@ squashed into multiple distinct change sets.
 
 ### Unit tests
 
-Unit test are ran with `cargo`:
+Unit test are ran with `cargo`.
+These cover code authored in Rust, but does not explicitly cover code authored
+in `<flox>/flox-bash/`.
 
 ```console
-$ flox develop flox --command "cargo test"
+$ flox develop flox --command 'cargo test';
 ```
 
 ### Integration tests
 
 Integration tests are written with `bats` and `expect`.
-They are located in the `./tests` folder.
+They are located in the `<flox>/tests` folder.
 To run them:
 
 ```console
-$ cargo build
-$ flox run .#flox-tests --flox ./target/debug/flox
+$ flox develop flox --command 'cargo build';
+$ flox shell '.#flox-tests' --command flox-tests --flox ./target/debug/flox;
```

Comment:
what's wrong with run here?
I dont think we even documented flox shell fwiw having that as part of our own testing seems counterintuitive

Reply:
`flox run` doesn't work here. I tried it several ways and the `flox run` arg parser will crash. Particularly try running with `flox run CMD -- ARGS`.

Reply:
you nee two `--` one for flox and one for nix...

Reply:
`flox run .#flox-tests -- -- --flox ./target/debug/flox`

Reply:
the other one is fun `flox run .#flox-tests -- -- --flox ./target/debug/flox -- ./tests/run.bats`

Reply:
how to work with three successive arg parsers ^^